apps/allocations/app/components/App/BudgetDetail.js

```diff
@@ -100,7 +111,7 @@ export default function BudgetDetail({ budget }) {
                     title="Budget"
                     large={displayCurrency(budget.amount, budget.token.decimals)}
                     small={budget.token.symbol}
-                    context="per 30 days"
+                    context={processTime(period.duration)}
```

Comment:
Is this the only place that needs to be changed? I was not sure if the Account period details component also needs some change

Reply:
Right now the ACCOUNT PERIOD DETAILS is correctly displaying the period.

So yes, this is the only place we need to update this right now.